### ISSUE TITLE
Hakar false

### ISSUE THREAD

**Issue by Cyanade** [MinecraftUserHAN1110/AntiCheatUnit#3]
Badly work on network of my large 300 players, this anticheat keeps falsing. Very correct no bad.

**Comment by Hydroxzz** [comment]
I agree sir, my serve has been using this anticheat. i have average 2-3 player but the antichet is keep banning all my player they were no hack. please fix. bad anticheat. i got ban for jumping

**Comment by Cyanade** [comment]
With you agree, jump = ban hacker work wurst client all day.
